dataland-internal-storage/src/test/kotlin/org/dataland/datalandinternalstorage/migrations/V2__MigrationTest.kt

```diff
@@ -0,0 +1,101 @@
+package org.dataland.datalandinternalstorage.migrations
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import db.migration.V2__MigrateEuTaxonomyNonFinancialsWithAbsoluteValues
+import org.flywaydb.core.api.migration.Context
+import org.json.JSONObject
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import java.sql.Connection
+import java.sql.ResultSet
+import java.sql.Statement
+
+@Suppress("ClassNaming")
+class V2__MigrationTest {
```

Comment:
Shouldn't the class name be V2__MigrateEuTaxonomyNonFinancialsWithAbsoluteValuesTest

Reply:
It doesn't matter too much since we get the uniqueness from the version number. But if you like, I can rename it.